cmd/vic-machine/inspect/inspect.go

```diff
@@ -144,7 +144,7 @@ func (i *Inspect) Run(clic *cli.Context) (err error) {
 		return errors.New("inspect failed")
 	}
 
-	vchConfig, err := executor.GetVCHConfig(vch)
+	vchConfig, err := executor.GetNoSecretVCHConfig(vch)
```

Comment:
the secret is query inside of method, instead of provided by the caller. for backward compatibility reason, if caller does not need the secret, we might leave it there.
so it's hard to pass in empty secret in this layer, but it's worth to make it better. will think about that in next PR, but not for this one. I prefer a quick fix for the problem.